js/page.js

```diff
@@ -22,6 +36,7 @@
     window.map.disable();
     window.mainPin.reset();
     window.filters.disable();
+
```

Comment:
https://github.com/htmlacademy-javascript/987137-keksobooking-19/blob/a9aae2f0f94f0251b8cba3265734ed3b7b28ff05/js/page.js#L7

При повторной активации страницы объявление загружаются заново, старые не нужны:
```suggestion
    allAdverts = null;
```